presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -339,6 +339,34 @@ public void shouldFailForIncorrectTrustStore()
         skipAfterTestWithContext();
     }
 
+    @Test(groups = {CLI, PROFILE_SPECIFIC_TESTS}, timeOut = TIMEOUT)
+    public void shouldPassForPasswordWithSpecialCharacters()
+            throws IOException, InterruptedException
+    {
+        ldapUserName = "UserWithSpecialPwd";
+        ldapUserPassword = "LDAP:Pass ~!@#$%^&*()_+{}|:\"<>?/.,';\\][=-`";
+        launchPrestoCliWithServerArgument("--catalog", "hive", "--schema", "default", "--execute", "select * from nation;");
+        assertThat(trimLines(presto.readRemainingOutputLines())).containsAll(nationTableBatchLines);
+    }
+
+    @Test(groups = {CLI, PROFILE_SPECIFIC_TESTS}, timeOut = TIMEOUT)
+    public void shouldPassForUserWithWhitespace()
+            throws IOException, InterruptedException
+    {
+        ldapUserName = "User WithSpace";
+        launchPrestoCliWithServerArgument("--catalog", "hive", "--schema", "default", "--execute", "select * from nation;");
+        assertThat(trimLines(presto.readRemainingOutputLines())).containsAll(nationTableBatchLines);
+    }
+
+    @Test(groups = {CLI, PROFILE_SPECIFIC_TESTS}, timeOut = TIMEOUT)
+    public void shouldFailForUserWithColon()
+            throws IOException, InterruptedException
+    {
+        ldapUserName = "UserWith:Colon";
+        launchPrestoCliWithServerArgument("--execute", "select * from hive.default.nation;");
+        assertTrue(trimLines(presto.readRemainingErrorLines()).stream().anyMatch(str -> str.contains("Illegal character ':' found in user name")));
+    }
+
```

Comment:
Can we combine these three tests into a single one? 
Say create a user with these credentials:
`ldapUserName=UserWith Space:SpecialPwd"`
`ldapUserPassword = .....`